lib/internal/fs/streams.js

```diff
@@ -86,6 +87,7 @@ function ReadStream(path, options) {
   this.pos = undefined;
   this.bytesRead = 0;
   this.closed = false;
+  this.isPerformingIO = false;
```

Comment:
Should we make this private? `this.#isPerformingIO`

Reply:
That only works for classes and we can’t turn this into a class rn

Reply:
ah, maybe a symbol then? It's minor. Just thinking out loud.

Reply:
Use a symbol?